src/core/Akka.NodeTestRunner/Program.cs

```diff
@@ -23,6 +30,14 @@ static void Main(string[] args)
             var testName = CommandLine.GetProperty("multinode.test-method");
             var displayName = testName;
 
+            var listenAddress = IPAddress.Parse(CommandLine.GetProperty("multinode.listen-address"));
```

Comment:
Grab the specified IP address and port from the commandline - if these arguments are not specified to `NodeTestRunner.exe` then the process will fail, unlike the `MultiNodeTestRunner.exe` binary which gets called directly by the end-user.